src/components/Kv/KvLightbox.vue

```diff
@@ -117,6 +112,18 @@ export default {
 		// Create and set our internal visibility property
 		visible() {
 			this.isShown = this.visible;
+			this.init();
+		}
+	},
+	mounted() {
+		this.isShown = this.visible;
+		this.init();
```

Comment:
I couldn't think of a good method name. Init function sets scroll lock, focus, focus lock, esc key listeners, but also tears them down.